GitUI/Editor/FileViewer.cs

```diff
@@ -768,7 +768,7 @@ void DetectDefaultEncoding()
 
         private static bool IsDiffView(ViewMode viewMode)
         {
-            return viewMode == ViewMode.Diff || viewMode == ViewMode.FixedDiff || viewMode == ViewMode.RangeDiff;
+            return viewMode is ViewMode.Diff or ViewMode.FixedDiff or ViewMode.RangeDiff;
```

Comment:
Example where pattern matching improves readability

Reply:
At the moment I'm not finding it easier, but perhaps with time I'll get used to it.